bisulfite/vcf2bed.cwl

```diff
@@ -0,0 +1,30 @@
+#!/usr/bin/env cwl-runner
+
+cwlVersion: v1.0
+class: CommandLineTool
+label: "Biscuit vcf2bed wrapper"
+baseCommand: ["/usr/bin/bsvcf2bed"]
+requirements:
+    - class: ShellCommandRequirement
```

Comment:
It doesn't look like this is currently doing anything that would require the `ShellCommandRequirement`.